changelog.md

```diff
@@ -239,6 +239,10 @@
 
 - Added `jsconsole.dir`, `jsconsole.dirxml`, `jsconsole.timeStamp`.
 
+- Added `jshttpclient` module for JavaScript that provides
```

Comment:
in PR msg, add something like: this migrates https://github.com/nim-lang/fusion/blob/master/src/fusion/js/jsxmlhttprequest.nim to stdlib if that's the intent